app/src/main/java/com/simplecity/amp_library/playback/MusicService.java

```diff
@@ -1014,7 +1014,6 @@ private void startForegroundImpl() {
                 Log.e(TAG, "startForeground should have been called, but song is null");
             }
         } catch (NullPointerException | ConcurrentModificationException e) {
-            Crashlytics.log("startForegroundImpl error: " + e.getMessage());
```

Comment:
We shouldn't have any empty catch statements, as they are very dangerous. We should instead log these to `Log.e` - for example:

`Log.e(TAG, "startForegroundImpl error: " + e);`

TAG is a static final string with the name of the class defined in this class.

Reply:
Ah, I thought I removed everything, Sorry about this!

Reply:
Got it will do that!